ember-core/src/main/scala/org/http4s/ember/core/Util.scala

```diff
@@ -17,12 +17,47 @@
 package org.http4s.ember.core
 
 import scala.concurrent.duration._
+import org.http4s.HttpVersion
+import org.http4s.Headers
+import org.http4s.Header
+import org.http4s.headers.Connection
+import org.typelevel.ci._
+import cats.data.NonEmptyList
+import java.util.Locale
 
 private[ember] object Util {
 
+  private[this] val closeCi = ci"close"
+  private[this] val keepAliveCi = ci"keep-alive"
+  private[this] val connectionCi = ci"connection"
+  private[this] val close = Connection(NonEmptyList.of(closeCi))
+  private[this] val keepAlive = Connection(NonEmptyList.one(keepAliveCi))
```

Comment:
We don't have a tradition of putting constants on headers with common values and/or low cardinality, but I don't know why not.  There's nothing Ember about these.

Reply:
Half the reason we're using strings here is so that we don't actually have to parse the headers (we can assume that we're looking at the raw headers since they've just been read from the socket)

Reply:
Oh, that's true.  Only the header values would be of much use.  (I bet blaze creates those uncached....)